hw/ip/prim/dv/prim_lfsr/prim_lfsr_testplan.hjson

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
We should fix dvsim so that it doesn't force us to add a testplan.

Reply:
I was actually able to remove the `testplan:` field in the sim_cfg.hjson file with no adverse effects.
Or, are you referring to the `tests:` specification at the bottom of the file?

Reply:
Yeah the `testplan` field in the sim cfg. If you were able to remove that line, then you can remove this file too?

Reply:
yeah, good point - I can delete this one as well.

Reply:
Are you still planning to delete this file?

Reply:
not as of yet - from our discussion a few days back, the testplan file was required for post-simulation creation of the results table.
If we think it's necessary, I can look into removing this dependency for IPs such as the `prim` modules so that the testplan is not required, but for now I think we should leave it in.